Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs

```diff
@@ -979,7 +979,7 @@ public virtual void CreateControls(object instance, IEnumerable<Tab> tabs)
                     Grid.SetIsSharedSizeScope(tabPanel, true);
                 }
 
-                var tabItem = new TabItem { Header = tab, Padding = new Thickness(4), Name = tab.Id };
+                var tabItem = new TabItem { Header = tab, Padding = new Thickness(4), Name = (tab.Id == null ? String.Empty : tab.Id) };

```

Comment:
Sure looks better. I am not fully used to C# yet, and in VB.NET we dont have the ?? operator.